queries/tlaplus/highlights.scm

```diff
@@ -160,3 +212,20 @@
 (proof_step_ref (level) @number)
 (proof_step_ref (name) @constant)
 (proof_step_ref ">" @punctuation.bracket)
+
+; Comments and tags
+(block_comment "(*" @comment)
+(block_comment "*)" @comment)
+(block_comment_text) @comment
+(comment) @comment
+(single_line) @comment
+(_ label: (identifier) @tag)
+(pcal_goto statement: (identifier) @tag)
+
+; Reference highlighting with the same color as declarations.
+; `constant`, `operator`, and others are custom captures defined in locals.scm
+((identifier_ref) @constant.builtin (#is? @constant.builtin constant))
+((identifier_ref) @operator (#is? @operator function))
+((identifier_ref) @parameter (#is? @parameter parameter))
+((identifier_ref) @variable.builtin (#is? @variable.builtin var))
+((identifier_ref) @include (#is? @include sequences))
```

Comment:
I've looked at our current implementation, you can use subcaptures, but the naming of the path must be exactly the same at in `locals`

Reply:
Ah, the `sequences` query is an extra, thanks for noticing.
Others (function, parameter, var, constant) are present in `locals`